13_0_Understanding_Your_Lightning_Setup.md

```diff
@@ -0,0 +1,33 @@
+# Chapter Thirteen: Understanding Your Lighting Setup
+
+> :information_source: **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+In this chapter you'll be ready to begin working with the `lightning-cli` command-line interface. But that first requires that you understand your c-lightning setup and its features, which is what will be explained with some examples and basic configuration.
+
+For this and future chapters, we presume that you have a VPS with c-lightning installed, running `lightningd` courtersy of [Bitcoin Standup Scripts](https://github.com/BlockchainCommons/Bitcoin-Standup-Scripts).
+We also presume that you are connected to testnet, allowing for access to bitcoins without using real funds. You can either do this by hand, per [§2.1: Setting up a Bitcoin-Core VPS by Hand](02_1_Setting_Up_a_Bitcoin-Core_VPS_by_Hand.md), with Bitcoin Standup at Linode.com, per [§2.2: Setting up a Bitcoin-Core VPS with Bitcoin Standup](02_2_Setting_Up_a_Bitcoin-Core_VPS_with_StackScript.md), or via other means, per [§2.3: Setting up a Bitcoin-Core Machine via Other Means](02_3_Setting_Up_Bitcoin_Core_Other.md).
+
```

Comment:
I've talking with Jo about c-lightning installation on BitcoinStandup Script and it's basically the typical installation aside from linux service and standup user.   Do you think we can include c-lightning installation steps here?

Reply:
Basically, if it's in BitcoinStandup, I'd prefer to lean on that. Just edit §2.2 to tell the user what they have to do if they want to install Lightning, to use it down the road in this section.

But, there's probably no harm in briefly including the instructions here too (or rather in 13.1) if we're still kind of in an in-between state, where it might not be merged into both of the Standup repos yet. So, if that's still the case, go ahead and put it in, but be aware that it's something that may be removed down the line, so it doesn't need a whole lot of effort.

Reply:
Done, i've added c-lightning instructions.

> Thanks for the updates, Javier, they look great. Here's what I think we still need:
> 
> (1) Decide if you want to include the installation instruction, and if so they probably belong in 13.1
> (2) In 13.3, when we create the channel, we need to help the readers in the creation of a machine2 (or node2 or whatever) that they're working with, and then clearly display the commands on each server 'machine1-c$', 'machine2-c$' will match the way I've described multiple machines in previous chapters and match your 'c'.
> (3) We also need to describe how that invoice is being created, presumably on the other machine, probably in a new section in advance of the current 13.4.

In 13.3 we're using only c-lightning commands,  no more lnd commands.    I'm updating 13.4 to show lnd commands clarifying which are lnd commands or c-lightning commands.